src/test/java/seedu/address/commons/util/ConfigUtilTest.java

```diff
@@ -76,8 +76,8 @@ private Config getTypicalConfig() {
         config.setAppTitle("Typical App Title");
         config.setLogLevel(Level.INFO);
         config.setUserPrefsFilePath("C:\\preferences.json");
-        config.setAddressBookFilePath("addressbook.xml");
-        config.setAddressBookName("TypicalAddressBookName");
+        config.setTaskManagerFilePath("addressbook.xml");
```

Comment:
Should we update the file path to `opus.xml` here? These are all hardcoded right now. I think it would be good to declare them as static variables.